insights/parsers/lspci.py

```diff
@@ -44,14 +44,14 @@ class LsPciDriver(CommandParser):
     Example:
 
         >>> lspcidriv = shared[LsPciDriver]
-        >>> pci_devs = lspcidriv.pci_dev_details.keys()
+        >>> lspcidriv.get_pci_devs()
```

Comment:
Since `get_pci_devs` is defined as `property`, the `()` should be removed.

Reply:
done